abzal.tankibayev/HomeWork02/readlog.js

```diff
@@ -0,0 +1,51 @@
+"use strict"
+var readline = require('readline-sync');
```

Comment:
Поменяйте здесь и далее декларации переменных на новые const/let
```suggestion
const readline = require('readline-sync');
```